packages/cozy-harvest-lib/test/components/__snapshots__/AccountModal.spec.jsx.snap

```diff
@@ -57,7 +57,7 @@ exports[`AccountModal with an account should display the AccountSelect & Content
     />
   </KonnectorModalHeader>
   <ModalContent>
-    <withClient(withI18n(KonnectorAccountTabs))
+    <withI18n(KonnectorAccountTabs)
```

Comment:
I don't understand this change. You didn't edit AccoutModal.jsx, it's weird, no?

Reply:
No but I changed `KonnectorAccountTabs` and removed the `withClient` there. Same for the other one :)